main.go

```diff
@@ -88,8 +89,59 @@ func main() {
 			if err := u.Put(ctx, summary); err != nil {
 				panic(err)
 			}
-		}
 
+			// If the Grype output is there, parse it and add a row for
+			// each vuln found
+			if summary.RawGrypeJSON != "" {
+				table := dataset.Table(GcloudTableVulns)
+				u := table.Inserter()
+				var output types.GrypeScanOutput
+				if err := json.Unmarshal([]byte(summary.RawGrypeJSON), &output); err != nil {
+					panic(err)
+				}
+				tmp := map[string]*types.Vuln{}
+				for _, match := range output.Matches {
+					var matchType string
+					for _, detail := range match.MatchDetails {
+						matchType = strings.TrimSuffix(detail.Matcher, "-matcher")
+						v := types.Vuln{
+							ScanID:        summary.ID,
+							Name:          detail.SearchedBy.Package.Name,
+							Installed:     detail.SearchedBy.Package.Version,
```

Comment:
This isn't a huge deal, but I'd suggest pulling these from `.artifact.name` and `.artifact.version` instead. This is the "source of truth" in terms of the matched package. As a bonus, this change might also mean we don't need the inner loop on `for _, detail := range match.MatchDetails`.

Reply:
Also there's a decent chance I'm not following this exactly right. I see we define another `types.Vuln` down below, too 🤔

Reply:
for some reason this does not result in the same number/combo of results displayed by the grype in a given scan. both appear to be needed to get the total

Reply:
I took another pass to simplify! please check again